shared/types/field/utils/myInfo.ts

```diff
@@ -0,0 +1,7 @@
+import { MyInfoPrefilledFormField, MyInfoFormField } from '../index'
+
+export const isMyInfoPrefilledFormField = (
```

Comment:
ah, should we rename this function to `hasExistingFieldValue` instead? I know currently the prop is only used by myinfo but it confused me. can probably also put in a more generic place

Reply:
i could shift it up 1 level into `field/utils` - wdyt? (also, renamed)